Sources/Commands/Destination.swift

```diff
@@ -140,14 +145,20 @@ public struct Destination {
   #if os(macOS)
     /// Returns the host's dynamic library extension.
     public static let hostDynamicLibraryExtension = "dylib"
+
+    /// Target subpath for the host system.
+    private static let hostTargetSubpath = "macosx-x86_64"
   #else
     /// Returns the host's dynamic library extension.
     public static let hostDynamicLibraryExtension = "so"
+
+    /// Target subpath for the host system.
+    private static let hostTargetSubpath = "linux-x86_64"
```

Comment:
The problem is, target includes deployment version on macOS (e.g. `x86_64-apple-macosx10.10`).